buildtools/shrinkwrap/shrinkwrap.js

```diff
@@ -22,27 +22,28 @@ var _ = require('lodash'),
 
 module.exports = function (grunt) {
 
-  function removeResolved(json) {
+  function removeResolvedAndFrom(json) {
```

Comment:
Interesting, I thought I had removed this, when I introduced install-with-shrinkwrap. Apparently, I did not. Do we want to keep this for the time being, since @lboyette-okta was seeing issues with install-with-shrinkwrap? @lboyette-okta were you able to make progress resolving extraneous modules when using installsw?